.github/workflows/python_bindings.yml

```diff
@@ -332,7 +332,11 @@ jobs:
           python -m pip install --upgrade pip
           # Note: if we choose to do ALL or nothing, remove  setuptools wheel twine (not needed unless trying to upload within this step)
           pip install requests packaging setuptools wheel twine
-          bindings_v=$(python ./python/module/find_pypi_tag.py --current)
+          if [[ "$GITHUB_REF" == *"refs/tags"* ]]; then
+            bindings_v=$(python ./python/module/find_pypi_tag.py --pypi)
+          else
+            bindings_v=$(python ./python/module/find_pypi_tag.py --current)
+          fi;
```

Comment:
You want to match whatever was computer in the previous step


https://github.com/NREL/OpenStudio/blob/7827af24a803e5adb686975f21cfa7c41c9565df/.github/workflows/python_bindings.yml#L115-L120

https://github.com/NREL/OpenStudio/blob/7827af24a803e5adb686975f21cfa7c41c9565df/python/module/CMakeLists.txt#L31-L35

Reply:
Makes sense that it wasn't finding the bindings as it was looking in the wrong repository.  looks good!